epilogue-runtime/src/main/java/edu/wpi/first/epilogue/logging/NTBackend.java

```diff
@@ -38,52 +40,92 @@ public class NTEpilogueBackend implements EpilogueBackend {
    *
    * @param nt the NetworkTable instance to use to send data to
    */
-  public NTEpilogueBackend(NetworkTableInstance nt) {
+  public NTBackend(NetworkTableInstance nt) {
     this.m_nt = nt;
   }
 
+  /** Creates a logging backend that sends information to NetworkTables. */
+  public NTBackend() {
+    this.m_nt = NetworkTableInstance.getDefault();
+  }
+
+  /**
+   * Creates a new NTBackend that is disabled whenever the disabledSupplier returns true.
+   *
+   * @param disabledSupplier the disable condition for NT logging
+   * @return a new NTBackend
+   */
+  public NTBackend disableWhen(BooleanSupplier disabledSupplier) {
```

Comment:
How is this better than just changing the backend on the fly?

```java
if (condition()) {
  Epilogue.configure(config -> {
    config.backend = new NullBackend();
  });
}
```

Reply:
It's more succint, and prevents you from having to cache different kinds of backends yourself.
For instance, if you only want NT logging enabled when fms is not attached, you would have to cache a file only backend, a file and nt backend, and use an addPeriodic call.

In addition, the configure() method accepts a lambda, and if it is called every loop, it will spawn a new object every loop as well